docs/index.md

```diff
@@ -127,9 +127,10 @@ After successfully installing Codacy on your cluster, you are now ready to perfo
 
 1.  Use a browser to navigate to the Codacy hostname previously configured on the file `values-production.yaml`.
 
-2.  Follow Codacy's onboarding process, which will guide you through the following steps:
+2.  Log in using your Git provider account. This automatically creates a Codacy administrator account with your credentials.
```

Comment:
I'm mentioning here that the first user to log into Codacy will become the Codacy administrator.